src/jit/compiler.h

```diff
@@ -10503,4 +10487,4 @@ BasicBlock* dFindBlock(unsigned bbNum);
 
 /*****************************************************************************/
 #endif //_COMPILER_H_
-/*****************************************************************************/
+/*****************************************************************************/
```

Comment:
It appears that this file has no newline.

Reply:
Good catch, will fix.